### PR TITLE
feat: REPLAT-5141/secondary one and four slice

### PR DESCRIPTION
**Pull request by ValGeorgiev**
New features:

- Secondary slice one and four (with white letters and black background)
image

- Tile N (with white letters and black background)
image

- Tile O (with white letters and black background)
image

What lefts:
1. The times logo needs to be added (will be done in a separate PR)

### CODE REVIEW

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 1]
```diff
@@ -16,13 +16,18 @@ const TileSummary = ({
   strapline,
   straplineStyle,
   style,
-  summary
+  summary,
+  summaryStyle,
+  flagColor,
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: This should be `flagColour` . We are using British English (unless it's a css/style prop, which forces US English :) )

File: packages/slice-layout/slice-layout.showcase.js [hunk 2]
```diff
@@ -240,12 +241,27 @@ export default {
...
+      component: () => (
+        <ScrollView>
+          <SecondaryOneAndFourSlice
+            renderSecondary1={() => <Support1 id="support1" />}
+            renderSupport1={() => <Support3 id="support3" />}
+            renderSupport2={() => <Support4 id="support4" />}
+          />
+        </ScrollView>
+      ),
+      name: "SecondaryOneAndFourSlice",
```
Comment (tuncaulubilge) [inline packages/slice-layout/slice-layout.showcase.js]: Looks like we got 3 tiles in here but the name say "SecondaryOneAndFour", so we might have a mix-up here

File: packages/edition-slices/src/slices/secondaryoneandfour/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { SecondaryOneAndFourSlice } from "@times-components/slice-layout";
+import { TileO, TileN } from "../../tiles";
+
+const SecondaryOneAndFour = ({ onPress, secondary1, support1, support2 }) => (
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/secondaryoneandfour/index.js]: shouldn't it take `support3, support4` as props too ?

File: packages/edition-slices/src/slices/secondaryoneandfour/index.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  <SecondaryOneAndFourSlice
+    renderSecondary1={() => <TileN onPress={onPress} tile={secondary1} />}
+    renderSupport1={() => <TileO onPress={onPress} tile={support1} />}
+    renderSupport2={() => <TileO onPress={onPress} tile={support2} />}
+    renderSupport3={() => <TileO onPress={onPress} tile={support2} />}
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/secondaryoneandfour/index.js]: title should be `support3`

File: packages/edition-slices/src/slices/secondaryoneandfour/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    renderSupport4={() => <TileO onPress={onPress} tile={support2} />}
+  />
+);
+
+SecondaryOneAndFour.propTypes = {
+  onPress: PropTypes.func.isRequired,
+  secondary1: PropTypes.shape({}).isRequired,
+  support1: PropTypes.shape({}).isRequired,
+  support2: PropTypes.shape({}).isRequired
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/secondaryoneandfour/index.js]: missing propTypes for `support3` and `support4`

File: packages/edition-slices/src/tiles/tile-n/index.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        style={styles.imageContainer}
+        uri={tile.article.leadAsset.crop11.url}
+      />
+      <TileSummary
+        flagColor={colours.functional.greyLabel}
+        headlineStyle={styles.headline}
+        labelColor={colours.functional.greyLabel}
+        style={styles.summaryContainer}
+        summary={tile.article.summary125}
+        summaryStyle={{ color: colours.functional.greyLabel }}
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/tile-n/index.js]: Can we pass in an object so no style is added in here ?

File: packages/slice-layout/src/templates/secondaryoneandfour/index.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  ];
+  return (
+    <View style={styles.container}>
+      <View style={styles.itemContainer}>
+        <View key={renderRowOne.props.id}>{renderRowOne}</View>
+        <View style={styles.itemColumnSeparator} />
+      </View>
+      <View style={styles.itemRowSeparator} />
+      <View key={renderFourRows[0].props.id}>{renderFourRows[0]}</View>
+      <View style={styles.itemRowSeparator} />
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryoneandfour/index.js]: We have a shared component `ItemRowSeparator` that you can use.

File: packages/slice-layout/src/templates/secondaryoneandfour/index.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    renderSupport1(),
+    renderSupport2(),
+    renderSupport3(),
+    renderSupport4()
+  ];
+  return (
+    <View style={styles.container}>
+      <View style={styles.itemContainer}>
+        <View key={renderRowOne.props.id}>{renderRowOne}</View>
+        <View style={styles.itemColumnSeparator} />
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryoneandfour/index.js]: use shared `ItemColSeparator`

File: packages/slice-layout/src/templates/secondaryoneandfour/styles.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    flex: 1
+  },
+  itemColumnSeparator: {
+    borderLeftWidth: 1,
+    marginVertical: spacing(2)
+  },
+  itemContainer: {
+    flexDirection: "row"
+  },
+  itemRowSeparator: {
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryoneandfour/styles.js]: can remove now

File: packages/slice-layout/src/templates/secondaryoneandfour/styles.js [hunk 10]
```diff
@@ -0,0 +1,23 @@
+import { spacing, colours } from "@times-components/styleguide";
+
+const styles = {
+  container: {
+    backgroundColor: colours.functional.darkSupplement,
+    flex: 1
+  },
+  itemColumnSeparator: {
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryoneandfour/styles.js]: can remove now

File: packages/slice-layout/src/templates/secondaryoneandfour/index.js (hunk 8 above)
Comment (ValGeorgiev) [reply packages/slice-layout/src/templates/secondaryoneandfour/index.js]: BTW we don't need this separator here, i will remove it. Thanks

**Comment by aronshamash** [comment]
looks good @ValGeorgiev

**Comment by aronshamash** [comment]
@ValGeorgiev needs a rebase before it will pass CI

**Pull request merged**